app/models/settings/community.rb

```diff
@@ -0,0 +1,21 @@
+module Settings
+  class Community < RailsSettings::Base
+    self.table_name = :settings_communities
+
+    # The configuration is cached, change this if you want to force update
+    # the cache, or call Settings::Community.clear_cache
+    cache_prefix { "v1" }
+
+    field :copyright_start_year,
+          type: :integer,
+          default: ApplicationConfig["COMMUNITY_COPYRIGHT_START_YEAR"] || Time.zone.today.year
+    field :community_description, type: :string
```

Comment:
The reason I didn't remove the `community_` prefix from these 3 attributes is that they have very generic names and our settings API controller currently handles things in a slightly hacky way so I'm trying to avoid name clashes.